index.html

```diff
@@ -32,7 +32,139 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
   <div class="arrow-flex display-flex">
     <span class="arrow">&or;</span>
 </div>
-</section>      
+</section>
+
+<!-- Recent Work Session -->
+<section class="container-recent">
+  <h2 class="recent-work">My Recent Works</h2>
+<div class="project-1 grid-project display-flex">
+  <div class="gray-container"></div>  
+  <div class="multi-post">
+    <h3 class="multi-text-color">Multi-Post Stories Gain+Glory</h3>
+  <ul class="language display-flex">
+    <li class="language-background">Ruby on rails</li>
+    <li class="language-background">css</li>
+    <li class="language-background" >JavaScript</li>
+    <li class="language-background">html</li>    
+  </ul>
+  <button class="see-project">See Project</button>
```

Comment:
- [ ] This button is highlighted as a point of reference. Kindly ensure ALL your buttons on the page has the styling for different effects: `enabled`, `hover` and `pressed` (active)

  ![ezea-btn](https://user-images.githubusercontent.com/36542986/164948971-8f376dbd-fecf-4c8c-8267-bf1808380401.jpg)

Reply:
good, fixed